src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECKCATEGORIES = 'bookstore/src/redux/categories/CHECK_CATEGORIES';
+
+export const checkcategories = {
+  type: CHECKCATEGORIES,
+  id: 0,
+};
+
+const checkReducer = (state = [], action) => {
```

Comment:
- Please, kindly create an initialState variable as required in the project requirement like this

` const initialState = [];`
`const checkReducer = (state = initialState, action) => {`

Reply:
Oki, it is done!